docs/components/AssemblyLine/answer_sets.md

```diff
@@ -75,12 +72,11 @@ set.
 To rename an answer set, first visit the answer set's information page:
 
 1. Find the answer set in the "My interviews" page and click to open
```

Comment:
Tangential nit: I realize you didn't write this, but doesn't clicking on the interview on the "My interviews" page open the interview? Am I wrong - does that instead open an intermediate page where you edit the answer set (for the interviews that have answer sets)?

Reply:
this is accurate as per the full context--when you choose "load answer set" inside a running interview, you get a list of answer sets. clicking one loads that set of answers into the running interview.